src/test/ui/macros/stringify.rs

```diff
@@ -790,26 +790,26 @@ fn test_ty() {
     assert_eq!(stringify_ty!([T]), "[T]");
 
     // TyKind::Array
-    assert_eq!(stringify_ty!([T; 0]), "[T; 0]");
+    assert_eq!(stringify_ty!([T; 0]), "[T ; 0]");
 
     // TyKind::Ptr
-    assert_eq!(stringify_ty!(*const T), "*const T");
-    assert_eq!(stringify_ty!(*mut T), "*mut T");
+    assert_eq!(stringify_ty!(*const T), "* const T");
+    assert_eq!(stringify_ty!(*mut T), "* mut T");
 
     // TyKind::Rptr
-    assert_eq!(stringify_ty!(&T), "&T");
-    assert_eq!(stringify_ty!(&mut T), "&mut T");
-    assert_eq!(stringify_ty!(&'a T), "&'a T");
-    assert_eq!(stringify_ty!(&'a mut T), "&'a mut T");
+    assert_eq!(stringify_ty!(&T), "& T");
+    assert_eq!(stringify_ty!(&mut T), "& mut T");
+    assert_eq!(stringify_ty!(&'a T), "& 'a T");
+    assert_eq!(stringify_ty!(&'a mut T), "& 'a mut T");
 
     // TyKind::BareFn
     assert_eq!(stringify_ty!(fn()), "fn()");
     assert_eq!(stringify_ty!(fn() -> ()), "fn() -> ()");
     assert_eq!(stringify_ty!(fn(u8)), "fn(u8)");
-    assert_eq!(stringify_ty!(fn(x: u8)), "fn(x: u8)");
+    assert_eq!(stringify_ty!(fn(x: u8)), "fn(x : u8)");
     #[rustfmt::skip]
-    assert_eq!(stringify_ty!(for<> fn()), "fn()");
-    assert_eq!(stringify_ty!(for<'a> fn()), "for<'a> fn()");
+    assert_eq!(stringify_ty!(for<> fn()), "for < > fn()");
```

Comment:
Changes in this file are expected because token pretty-printer is now used instead of AST pretty-printer.
This particular line is actually a bugfix.